test/test_sparse_csr.py

```diff
@@ -1270,9 +1272,10 @@ def test_sparse_csr_consistency(self, device, dtype, op):
             if sample.input.ndim != 2:
                 continue
 
-            expected = op(sample.input)
+            expected = op(sample.input, *sample.args, **sample.kwargs)
+
             assert torch.is_tensor(expected)
-            output = op(sample.input.to_sparse_csr())
+            output = op(sample.input.to_sparse_csr(), *sample.args, **sample.kwargs)
```

Comment:
All tests still pass without this change, of course, it's good to have it more generic for future changes.